seedgen/config.cpp

```diff
@@ -89,12 +90,13 @@ ConfigError Config::loadFromFile(const std::string& filePath, const std::string&
 
         settings.plandomizerFile = Utility::get_app_save_path() +  "plandomizer.yaml";
     #else
-        std::string baseTemp, outTemp;
+        std::string baseTemp, outTemp, plandoTemp;
         GET_FIELD_NO_FAIL(preferencesRoot, "gameBaseDir", baseTemp)
-        gameBaseDir = baseTemp;
+        gameBaseDir = baseTemp == "" ? u"" : Utility::Str::toUTF16(baseTemp);
         GET_FIELD_NO_FAIL(preferencesRoot, "outputDir", outTemp)
-        outputDir = outTemp;
-        GET_FIELD_NO_FAIL(preferencesRoot, "plandomizerFile", settings.plandomizerFile)
+        outputDir = outTemp == "" ? u"" : Utility::Str::toUTF16(outTemp);
+        GET_FIELD_NO_FAIL(preferencesRoot, "plandomizerFile", plandoTemp)
+        settings.plandomizerFile = plandoTemp == "" ? u"" : Utility::Str::toUTF16(plandoTemp);
```

Comment:
Should work without the ternaries after 05980b36fc1452777efd08e9f6176995d5a06a50